ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -50,66 +50,30 @@
         placeholder='10.0.0.1, 10.0.0.2, 10.0.0.3'
       />
     {{/if}}
+    <Form::Field::ListWrapper
+      @layout='horizontal'
+      @isOptional={{true}}
```

Comment:
Is it an optional field?

Reply:
@DhariniJeeva good call they are not optional.  Change in next commit